packages/docbook-style-dsssl/SPECS/docbook-style-dsssl.spec

```diff
@@ -33,32 +33,32 @@ cp %{SOURCE1} Makefile
 
 %install
 DESTDIR=$RPM_BUILD_ROOT
-make install BINDIR=$DESTDIR/usr/bin DESTDIR=$DESTDIR/usr/share/sgml/docbook/dsssl-stylesheets-%{version} MANDIR=$DESTDIR%{_mandir}
+make install BINDIR=$DESTDIR/usr/sgug/bin DESTDIR=$DESTDIR/usr/sgug/share/sgml/docbook/dsssl-stylesheets-%{version} MANDIR=$DESTDIR%{_mandir}
```

Comment:
Nitpick: Why not use `%{_prefix}` or `%{_bindir}` or `%{_datadir}` for these instead of hard paths